### ISSUE TITLE
accessibility issue 4 (medium)-The same link text is used for links going to different destinations. 

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#237]
The same link text is used for links going to different destinations. Users might not know the difference if they are not somehow explained.  ·       There are two “Yale Center for British Art” links (one at the top of the page that goes to https://britishart.yale.edu/and one in the footer that goes to https://collections.britishart.yale.edu/·       There are two “Reference Library” links (one used as a filter in the search and one in the footer).  One goes to https://collections.britishart.yale.edu/?f%5Bcollection_ss%5D%5B%5D=Reference+Library and the other goes to https://britishart.yale.edu/reference-library-and-photograph-archive
--

**Comment by yulgit1** [comment]
First point addressed in https://github.com/ycba-cia/blacklight-collections2/commit/7f9336d53ab1d62d3be099193808991897042b79

Second point, the context should be clear enough for the user to distinguished a link to the reference library page in the footer from a facet limiting the collections search to materials within the reference library.  Keeping as is.
